src/platform/common/namespaces.ts

```diff
@@ -0,0 +1,55 @@
+import * as path from '../../platform/vscode-path/path';
+export * from './constants';
+
+export namespace DataFrameLoading {
+    export function SysPath(rootDir: string) {
```

Comment:
This is good but I think all of these should take a URI and return a URI. We shouldn't be doing path manipulation with  path entries. Path entries are always posix, path.join is OS specific.

You'd use the urlPath functions instead to concat stuff together. See 'src/platform/vscode-path/resource'.

Reply:
Rich's recent widgets PR has some good examples of this I think to look at for the path => Uri conversion.

Reply:
also id suggest renaming the file. namespace is too ambiguous. namespace are just a collection names. 
eg rename to script utils or something that's a more descriptive. as this no longer contains constants, but contains done utility functions to return psths